lib/services/components.js

```diff
@@ -446,7 +446,7 @@ function getPossibleTemplates(name, templateName) {
 /**
  * Get the first template found for a particular component
  * @param {string} name  i.e., 'article', 'paragraph', 'content', 'my-layout'
- * @param {string} templateName  i.e., 'template', 'sitemap', 'apple', 'facebook'
+ * @param {string} templateName  i.e., 'template', 'apple', 'facebook'
```

Comment:
Dunno. The other probably shouldn't have been in brackets in the first place because that var can't be an array. Our commenting around functions is kinda all over the place. I've removed the brackets in a new commit.